inst/rmarkdown/templates/mdpi/resources/template.tex

```diff
@@ -227,52 +267,52 @@
 \end{tabular}}
 $endif$
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Optional
 $if(appendix)$
 \input{"$appendix$"}
 $endif$
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-% Citations and References in Supplementary files are permitted provided that they also appear in the reference list here.
+\end{paracol}
```

Comment:
What is this for ? Should it be a `\begin{paracol}` somewhere ? 

Maybe I missed it.